src/components/Days-Catches/DisplayDays.js

```diff
@@ -22,11 +124,69 @@ export default function DisplayDays({
               />
             </li>
           ))
-        : '...loading cards...'}
-    </CardsList>
+        ) : (
+          <div>
+            <Hint>...Loading data...</Hint>
```

Comment:
Just for the future: framer-motion is a cool library for animated everything including loading placeholder stuff